libs/ui/src/examples/TanstackQuery/E3_MultiFetchQuery.tsx

```diff
@@ -49,13 +53,18 @@ export const MultiFetchQuery = () => {
 
   const blogsQuery = useBlogsQuery(selectedAuthorId)
   const authorsQuery = useAuthorsQuery()
-
-  console.log('authorsQuery', authorsQuery)
+  console.log(authorsQuery.data?.data)
 
   return (
     <Stack>
       <Select placeholder="Select author">
         {/* TODO: List all the authors and set the selectedAuthorId with onChange method */}
+        {authorsQuery.data?.data.map(author => (
+          // eslint-disable-next-line react-hooks/rules-of-hooks
+          <option onChange={() => setSelectedAuthorId(author.id)}>
```

Comment:
You can use onChange in `Select` element.

`onChange(e => setSelectedAuthorId(e.target.value))`

```suggestion
          <option key={author.id} value={author.id}>
```